apps/alerts/config/test.exs

```diff
@@ -1,3 +1,5 @@
 use Mix.Config
 
-config :alerts, api_mfa: {JsonApi, :empty, []}
+config :alerts,
+  api_mfa: {JsonApi, :empty, []},
```

Comment:
question: is `api_mfa` used anywhere?

Reply:
@paulswartz It's retrieved in `Alerts.Supervisor` to assign the module/function/args to `Alerts.Cache.Fetcher`.